src/Browsershot.php

```diff
@@ -58,11 +59,13 @@ public static function url(string $url)
 
     /**
      * @param string $html
+     * @param array $safeDomains
      *
      * @return static
      */
-    public static function html(string $html)
+    public static function html(string $html, $safeDomains = [])
     {
+        (new static())->jsLinksInWhitelist($html, $safeDomains);
```

Comment:
Right now, you're using two names for the same concept: `safeDomains` and `whiteList`. Let's make it clear by using the inclusive term `allowList` for both.